### PR TITLE
Fix/verbose

### PR DESCRIPTION
**Pull request by mike0sv**

### CODE REVIEW

**Review by omesser — APPROVED** [review]

File: tests/api/test_commands.py [hunk 1]
```diff
@@ -240,3 +241,12 @@ def test_build_lazy(model_meta, tmp_path):
     build(
         "pip", model_meta, target=str(tmp_path / "build"), package_name="lol"
     )
+
+
+def test_serve_no_signature(model):
+    meta = MlemModel.from_obj(model)
+    with pytest.raises(
+        MlemError,
+        match="Cannot create interface from model with uspecified signature. Please re-save it and provide sample_data argument",
```
Comment (aguschin) [inline tests/api/test_commands.py:250]:
```suggestion
        match="Cannot create interface from model with unspecified signature. Please re-save it and provide `sample_data` argument",
```

File: mlem/runtime/interface.py [hunk 2]
```diff
@@ -189,6 +189,22 @@ def load(self, uri: str):
...
+        if any(
+            isinstance(a, UnspecifiedDataType)
+            for method in value.methods.values()
+            for a in method.args
+        ) or any(
+            isinstance(method.returns, UnspecifiedDataType)
+            for method in value.methods.values()
+        ):
+            raise MlemError(
+                "Cannot create interface from model with uspecified signature. Please re-save it and provide sample_data argument"
```
Comment (aguschin) [inline mlem/runtime/interface.py:204]:
```suggestion
                "Cannot create interface from model with unspecified signature. Please re-save it and provide `sample_data` argument"
```

**Comment by aguschin** [comment]
1. Are you trying to replace https://github.com/iterative/mlem/pull/414 ?
2. You included changes from https://github.com/iterative/mlem/pull/468 by occasion

**Comment by mike0sv** [comment]
1. yep
2. let's merge #468 first then

**Review by aguschin — APPROVED** [review]

**Pull request merged**